lib/brains/forms.py

```diff
@@ -110,7 +110,10 @@ def format_descriptor(self, name):
         return 'Mozilla*{}'.format(name)[0:22]
 
     def get_name(self, plan_id):
-        if plan_id in products:
+        # The only way I can get the mock to stick is to do a late import.
+        # If we can fix that we can move this up.
+        from payments_config import products
```

Comment:
did you try `import payments_config.products` at the top and then patching `payments_config` ? That should work. Not sure if it would be any prettier though. You could also try patching `get()` on `products` directly which would probably work as well.